### ISSUE TITLE
Test evaluation done on training set?

### ISSUE THREAD

**Issue by basveeling** [noahfl/densenet-sdr#4]
Hey guys, I thought your paper was very interesting and I took a shot at running your code (thanks for sharing!). I noticed a suspicious line flagged by my IDE as having no effect:
https://github.com/noahfl/densenet-sdr/blob/3b51b95e0752cf3867fd7037897a385aeb1365fb/models/dense_net.py#L638
I hope I'm wrong, but I think the lines where you load your test set don't affect the session graph, and thus it seems that you're reporting validation scores on the training set. I've added an extra validation loop that's explicitly evaluating the training set and it reports roughly the same values.

**Comment by noahfl** [comment]
Wow, thanks for bringing this to my attention. I'm pretty new to TensorFlow so this is definitely my mistake. I'm out of the office for the week but I'm trying to push out a fix for this ASAP. In the meantime I'll update the paper to reflect that we're reporting training error and not testing error for the time being so as not to mislead anyone.

**Comment by jph00** [comment]
Great job all around - this is what open science is all about! 😄

**Comment by noahfl** [comment]
Okay, I've pushed a [potential fix for this here](https://github.com/noahfl/densenet-sdr/blob/6a9beb19dcbc0d92fcd9561ccee786497ee61c99/models/dense_net.py#L634). Like I said, I'm out of the office for the week but I'll try and keep on top of this. If anyone finds anymore discrepancies please don't hesitate to let me know. I'm the only person in my lab who works with TensorFlow so it's nice to have some fresh eyes on this.

As it stands the current results in the paper are valid for the purpose of comparison of _training_ performance, but I'll be rerunning these (provided the code I've pushed passes scrutiny) to collect test data and updating the paper and the repo as necessary.

**Comment by noahfl** [comment]
For anyone that's curious, here was the issue: while converting the original code from using ```feed_dict```s to using QueueRunners and ```tf.train.shuffle_batch```, I failed to properly set up the test function with its own QueueRunner and batch generator. Now that it seems to be fixed I'm rerunning the benchmarks and updating the README with the new figures. Once that's all finished we can update the paper with the new figures and be on our merry way.

**Comment by Madder** [comment]
Any update on the new results (on the test set) ?
